app/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,12 @@
     <string name="settings">Settings</string>
     <string name="update">Update</string>
 
+    <string name="whenCreateAPost">When someone creates a post</string>
+    <string name="createPostNotification">This will send a notification when someone creates a post</string>
+    <string name="whenBookmarksAPost">When someone bookmarks a post</string>
+    <string name="bookmarksPostNotification">"This will send a notification when someone bookmarks a post   "</string>
```

Comment:
The string names for the preferences can be based on the preference identifiers themselves:

```
- newPostNotification
- newPostNotificationHint
- newBookmarkNotification
- newBookmarkNotificationHint
```